packages/firestore/test/unit/specs/spec_builder.ts

```diff
@@ -1095,9 +1093,12 @@ export class SpecBuilder {
   private addQueryToActiveTargets(
     targetId: number,
     query: Query,
-    resumeToken?: string,
-    readTime?: TestSnapshotVersion
+    resume?: ResumeSpec
   ): void {
+    if (!(resume?.resumeToken || resume?.readTime) && resume?.expectedCount) {
+      fail('ExpectedCount is present without a resumeToken or readTime.');
```

Comment:
nit: "ExpectedCount" -> "expectedCount" so that the character cases are consistent.